community-operators/postgres-operator/postgresql.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: postgresql
```

Comment:
Rename the directory to match the packageName

Reply:
I renamed the directory to match the package name.